packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -105,7 +86,7 @@ export class DevServerBuilder implements Builder<DevServerBuilderOptions> {
         const serverAddress = url.format({
           protocol: options.ssl ? 'https' : 'http',
           hostname: options.host === '0.0.0.0' ? 'localhost' : options.host,
-          port: options.port.toString(),
+          port: (options.port || 4200).toString(),
```

Comment:
Why not use the defaults in the schema?

Reply:
The type is optional, so typescript complains here. I need to figure out how to better do this.